jenkins/swift-functional-tests/20-install-openstack.sh

```diff
@@ -9,20 +10,37 @@ enable_service key mysql s-proxy s-object s-container s-account
 SCREEN_LOGDIR="\${DEST}/logs"
 EOF
     cp jenkins/${JOB_NAME%%/*}/extras.d/55-swift-sproxyd.sh devstack/extras.d/55-swift-sproxyd.sh
+    if [[ $DEVSTACK_BRANCH == "stable/icehouse" ]]; then
+        # Workaound depencies version conflict :
+        # last python-glanceclient (0.17 at the time of writing), wich gets installed by default,
+        # requires keystoneclient >= 1.0.0
+        # whereas keystone requires keystoneclient <= 0.11.2
+        # python-glanceclient >= 0.13.1 is required by openstackclient 0.4.1
+        sudo pip install python-glanceclient==0.13.1
+    fi
     ./devstack/stack.sh
 }
 
+function ubuntu_common {
+    wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python;
```

Comment:
The latest  version number of the moment is hardcoded in that script.
By copying that file somewehere, we won't use long the last version of setuptools, which is the whole point.